### DISCUSSION TITLE
Error communicating with PDF uploaded into Pinecone

### DISCUSSION THREAD

**Discussion by pealingthunder** [FlowiseAI/Flowise#461] [Q&A]
Hey folks, 
I have created a flow to upload PDF and that worked fine - I can see and chat with the data in pinecone from that flow.
However every time I submit a question, it loads the PDF data.

So as recommended, I made another flow just to Chat with the existing Pinecone index.

When I create a flow to Load the existing index - it does not work.
**I keep getting this error:  "Request failed with status code 401"**

I ended up getting a paid Pinecone account to see if it would fix the issue - and it did not.

All my params are correct. API / Environment / Index
I noticed many people were having issues with environment "us-east4-gcp" - On the Pinecone forum it seems like some people fixed it by hardcoding the environment.

What could be wrong here in my Flowise config?

Thanks!

**Comment by HenryHengZJ** [comment]
Are you using API or Flowise UI Chat?
I'd say try create a new pinecone API key, you shouldn't need to have pinecone paid version

**Reply by pealingthunder** [reply]
I am using the Flowise UI Chat,  the Pinecone Load Existing Index (with API Key / Env. / Index)

I was testing a a couple other Chains.
![image](https://github.com/FlowiseAI/Flowise/assets/138264151/3d79ac09-c1f7-4705-bbfd-1c8da89a2533)

Still not working
